### PR TITLE
Remove ?T print style

### PR DESCRIPTION
**Pull request by cjprybol**
https://github.com/JuliaData/Nulls.jl/commit/b1a555d119a7537abe6e8f57451c19f7caeece46##commitcomment-23842283 and https://github.com/JuliaData/DataTables.jl/pull/83#discussion_r134702649

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -131,5 +131,5 @@ using Base.Test, Nulls
     @test Nulls.T(Any) == Any
     io = IOBuffer()
     show(io, Union{Float64, Null})
-    @test String(take!(io)) == "?Float64"
+    @test String(take!(io)) == "Union{Nulls.Null, Float64}"
```
Comment (quinnj) [inline test/runtests.jl]: Don't worry about that; just make the test say `@test String(take!(io)) in ("Union{Nulls.Null, Float64}", "Union{Float64, Nulls.Null}")`. There's a ton of logic that goes into the Union type sorting that's in 0.7 only.

Comment (ararslan) [inline test/runtests.jl]: You know it'll be one order or the other regardless of the version, so I'd do
```julia
@test String(take!(io)) in ["Union{Nulls.Null, Float64}", "Union{Float64, Nulls.Null}"]
```

Comment (ararslan) [inline test/runtests.jl]: Jacob ninja'd my recommendation

Comment (quinnj) [inline test/runtests.jl]: Actually. just take this test out for now; it's not like we're actually testing anything here anyway.

**Pull request closed without merging**